---
layout: page
title: Research
permalink: /research/
---

<b>Publication</b><br>
<strong><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7372605/pdf/10.1177_1176934320915707.pdf" target="_blank">Prediction of RNA Methylation Status From Gene Expression Data Using
Classification and Regression Methods</a></strong><br><br>
<p style="text-align:justify"><i>Abstract: </i>Abstract:RNA N6-methyladenosine (m6A) has emerged as an important epigenetic modification for its role in regulating the stability, structure, processing, and translation of RNA. Instability of m6A homeostasis may result in flaws in stem cell regulation, decrease in fertility, and risk of cancer. To this day, experimental detection and quantification of RNA m6A modification are still time-consuming and labor-intensive. There is only a limited number of epitranscriptome samples in existing databases, and a matched RNA methylation profile is not often available for a biological problem of interests. As gene expression data are usually readily available for most biological problems, it could be appealing if we can estimate the RNA methylation status from gene expression data using in silico methods. In this study, we explored the possibility of computational prediction of RNA methylation status from gene expression data using classification and regression methods based on mouse RNA methylation data collected from 73 experimental conditions. Elastic Net-regularized Logistic Regression (ENLR), Support Vector Machine (SVM), and Random Forests (RF) were constructed for classification. Both SVM and RF achieved the best performance with the mean area under the curve (AUC)=0.84 across samples; SVM had a narrower AUC spread. Gene Site Enrichment Analysis was conducted on those sites selected by ENLR as predictors to access the biological significance of the model. Three functional annotation terms were found statistically significant: phosphoprotein, SRC Homology 3 (SH3) domain, and endoplasmic reticulum. All 3 terms were found to be closely related to m6A pathway. For regression analysis, Elastic Net was implemented, which yielded a mean Pearson correlation coefficient=0.68 and a mean Spearman correlation coefficient=0.64. Our exploratory study suggested that gene expression data could be used to construct predictors for m6A methylation status with adequate accuracy. Our work showed for the first time that RNA methylation status may be predicted from the matched gene expression data. This finding may facilitate RNA modification research in various biological contexts when a matched RNA methylation profile is not available, especially in the very early stage of the study.</p>
<br>
<b>Mathemtical Modeling Contest</b><br>
<strong><a href="https://github.com/xvehao/xvehao.github.io/blob/master/file/2017-xjtlu-mathematic%20copy.pdf" target="_blank">title</a></strong><br><br>
<p style="text-align:justify"><i>Abstract: </i>abstract.</p>
<br>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7372605/pdf/10.1177_1176934320915707.pdf" target="_blank">PDF.</a>
<b>Poster</b><br>
